docs/guides/src/main/server/logging.adoc

```diff
@@ -429,6 +430,35 @@ With this configuration applied, start keycloak using `start-dev` or `start`.
 ==== Kibana: See the results
 Open http://localhost:5601 to reach the Kibana dashboard. The exact configuration of a good monitoring dashboard is out of scope for this guide. The easiest way to find out if logs sent by Keycloak are delivered to Kibana is to open the http://localhost:5601/app/kibana#/dev_tools/console?_g=()[Dev Tools] and execute the default `match_all` query. The logs should appear in the result field.
 
+=== Configure a different log level for the GELF logger
+To keep log storage costs and verbosity low, it is often wanted to only store a subset of the verbose application logs inside a centralized log management system. To configure Keycloak to use a different log level for the logs you want to ingest, use the following configuration:
+
+[source, conf]
+----
+log=console,gelf
+log-gelf-level=<desired-log-level>
+...
+----
+
+.Example
+To only see occurred log levels of warn and above in your centralized logging stack, but still see INFO level logs on the applications console logs, use the following configuration:
+
+[source, conf]
+----
+log=console,gelf
+log-level=INFO
+log-gelf-level=warn
+...
+----
+
+Looking at your ingested logs, you will see only messages of level warn or above will appear.
+
+Keep in mind that `--log-level` is setting the leading log level, so for example when you invoke the following command:
+
+<@kc.start parameters="--log=console,gelf, log-level=error, log-gelf-level=info"/>
+
+nothing below the error level will be sent to your logging stack.
```

Comment:
```suggestion
nothing below the error level will be sent to your logging stack, including GELF in this case.
```

Reply:
tbh, this suggestion doesn't make much sense to me. Can you elaborate?

Reply:
@DGuhr Sorry, missed your comment. I adjusted my suggestion above hopefully to make it more clear. But maybe I just misunderstood it. :)

Reply:
tbh i think it is redundant, because the sentence below states that nothing below error level will be sent to your (distributed) logging stack. But when it makes it clearer for you, it's fine for me. Will commit the suggestion :)